typecats/attrs_shim.py

```diff
@@ -100,11 +101,12 @@ def wrap(cls):
             field_transformer,
         )
 
+        # This should be the only change in here.
```

Comment:
Sorry for the confusion- the PR doesn't show a diff because the "change" was already there. That is the change from the upstream function (linked in the comment above)